### PR TITLE
testing/py-confluent-kafka: new aport

### PR DESCRIPTION
**Pull request by gena01**

### CODE REVIEW

**Comment by TBK** [comment]
@gena01 Please make this a py3 only `testing/py3-confluent-kafka`. We are in the process of removing py2.

**Comment by gena01** [comment]
@TBK you have any links/examples?

**Comment by TBK** [comment]
@gena01 https://wiki.alpinelinux.org/wiki/Python_package_policies#Python_3_only & from commit d8ab4fdef8eb6d1085a7ec69e586e5609434c762

File: testing/py-confluent-kafka/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+makedepends="py3-setuptools python3-dev librdkafka-dev>=1.0.1"
+source="$pkgname-$pkgver.tar.gz::https://github.com/confluentinc/confluent-kafka-python/archive/v$pkgver.tar.gz"
+builddir="$srcdir/$_pkgname-$pkgver"
+
+build() {
+	python3 setup.py build
+}
+
+package() {
+	mkdir -p "$pkgdir"
```
Comment (maxice8) [inline testing/py-confluent-kafka/APKBUILD]: no need to run this if you are running the command below it

**Comment by gena01** [comment]
Going to be updating this. Will try to pull #8763 in.

**Comment by maxice8** [comment]
Can be closed, merged with minor changes (fix package directory and the commit title)

**Pull request closed without merging**